### PR TITLE
Move Repo to new ValidationLayers/Tools/Headers Submodules

### PR DESCRIPTION
**Pull request by mark-lunarg**
Builds on Windows and Linux. 

None of the Vulkan-Tools parts have been wired in yet (vulkaninfo, cube, mock ICD) so most of the tests will not run.  Feel free to jump in if you're nonplussed.

### CODE REVIEW

**Comment by mark-lunarg** [comment]
@joey-lunarg, @mikew-lunarg, there's likely some additional work to get the more complicated vktrace and devsim test scripts happy.  Please take a look, test on a couple of platforms, and let's try to get this in ASAP.

**Comment by mark-lunarg** [comment]
Just rebased onto master, BTW.

**Comment by mark-lunarg** [comment]
@davidlunarg, @jeremyk, @tobine, you guys please give it a checkout also.

File: BUILD.md [hunk 1]
```diff
@@ -23,11 +23,31 @@ sudo apt-get install git cmake build-essential bison libx11-xcb-dev libxkbcommon
 sudo apt-get install wget autotools-dev libxcb-keysyms1 libxcb-keysyms1-dev
 # If performing 32-bit builds, you will also need:
 sudo apt-get install libc6-dev-i386 g++-multilib
+
```
Comment (mikew-lunarg) [inline BUILD.md]: Move the code markdown terminator here from line 46

**Comment by mark-lunarg** [comment]
Rebased and pushed to Master.  Call if you have any questions!

**Pull request merged**